docs/UserGuide.md

```diff
@@ -440,79 +457,113 @@ This section guides you on how to find or filter students of your choice in TAB.
-Lesson Homework  | hw/LESSON_HOMEWORK_KEYWORDS | Students who have homework that contains all the keywords
-
-Notes about the find condition:
-
-* The find condition indicates that a student is only considered a match when `all`, `any` or `none`
-  of the fields which you are searching for match the student.<br>
-  e.g. 
-    * `find n/John t/math cond/all` will return students with both the name `John` and the tag `math`.
-    * `find n/John t/math cond/any` will return students with only the name `John`, or only the tag `math`, or both.
-    * `find n/John t/math cond/none` will return students without the name `John` and the tag `math`.
-    
-* The find condition is optional and defaults to `all` if not specified. <br>
-  e.g. `find date/10 Oct 2021 time/1000-1400` will return students with lessons that occur between `1000-1400` on `10 Oct 2021`.
-  
-Notes about search keywords:
+Format: `find [cond/{all | any | none}] [t/TAG_KEYWORD]…​ [n/NAME_KEYWORDS] [a/ADDRESS_KEYWORDS] [p/PHONE_KEYWORDS] [e/EMAIL_KEYWORDS] [pp/PARENT_PHONE_KEYWORDS] [pe/PARENT_EMAIL_KEYWORDS] [sch/SCHOOL_KEYWORDS] [stream/ACAD_STREAM_KEYWORDS] [lvl/ACAD_LEVEL_KEYWORDS] [r/REMARK_KEYWORDS] [subject/SUBJECT_KEYWORDS] [date/START_DATE] [time/TIME_RANGE] [cancel/CANCELLED_DATE] [rates/LESSON_RATE_KEYWORDS] [hw/HOMEWORK_KEYWORDS]`
 
-* You must provide at least one field to search.<br>
-  e.g. entering just `find` or `find cond/any` alone is not a valid command. You need to include the fields you wish to search for.
+* You must specify at least one field to search for, and provide at least one keyword.<br>
+  e.g. `find cond/any` or `find n/` are invalid commands.
 
-* You must provide at least one keyword to search for.<br>
-  e.g. entering just `find n/` alone is not a valid command as the keyword is empty.
+##### Search by tags
+* You can search by multiple tags. e.g. `t/paid t/new`
+* One tag parameter can only have one keyword. e.g. `t/paid new` is invalid.
+* The search is case-insensitive. e.g. keyword `new` will match `NEW`.
+* Only whole words will be matched. e.g. keyword `paid` will not match `unpaid`
 
-* Tags must only have one keyword.<br>
-  e.g. `find t/zoom math` is invalid. To search by multiple tags, you can do `find t/zoom t/math`.
+Example:
+* `find t/unpaid t/New` will return all students with both of the tags `UNPAID` and `NEW`.
 
-* The search is case-insensitive.<br>
-  e.g. keyword `hans` will match `Hans`.
+##### Search by other student fields
+* You can specify one or more keywords for each parameter. e.g. `find n/Amad Ali`.
+* A field needs to contain all specified keywords to be matched.<br>
+  e.g. keywords `Amad Ali` will not match `Amad`, but it will match `Amad bin Ali`.
+* If you specify multiple of the same prefix, only the last prefix will be used for the search of that field.
+* A keyword can match a word partially. e.g. keyword `uni` will match `university`.
+* The search is case-insensitive.
```

Comment:
Super minor thing but line 481,482 not in same order as line 470,471 ahaha

Reply:
what do you mean by this?

Reply:
In 470,471, the case sensitivity point comes after how the word is matched